src/Components/Components/src/PublicAPI.Unshipped.txt

```diff
@@ -25,3 +31,5 @@ override Microsoft.AspNetCore.Components.EventCallback.GetHashCode() -> int
 override Microsoft.AspNetCore.Components.EventCallback.Equals(object? obj) -> bool
 override Microsoft.AspNetCore.Components.EventCallback<TValue>.GetHashCode() -> int
 override Microsoft.AspNetCore.Components.EventCallback<TValue>.Equals(object? obj) -> bool
+virtual Microsoft.AspNetCore.Components.RenderTree.Renderer.AddPendingTask(Microsoft.AspNetCore.Components.Rendering.ComponentState? componentState, System.Threading.Tasks.Task! task) -> void
+virtual Microsoft.AspNetCore.Components.RenderTree.Renderer.CreateComponentState(int componentId, Microsoft.AspNetCore.Components.IComponent! component, Microsoft.AspNetCore.Components.Rendering.ComponentState? parentComponentState) -> Microsoft.AspNetCore.Components.Rendering.ComponentState!
```

Comment:
These two new `protected virtual` methods are what allows `EndpointHtmlRenderer` to deal with everything to do with tracking streaming SSR on a per-subtree basis without mixing those concerns into the `Renderer` base class.

Normally I'm quite reluctant to add any public API to `Renderer` but I think adding these reasonably general-purpose `protected virtual` methods is the least disruptive thing we could do overall.